lib/Services/firebaseStroageService.dart

```diff
@@ -9,22 +9,32 @@ class FirebaseStorageServices{
   File _image;
   BuildContext _context;
 
-  FirebaseStorageServices(this._image,this._context);
-  
+  static final FirebaseStorageServices _singleton = new FirebaseStorageServices._internal();
+
+  FirebaseStorageServices._internal();
+
+  factory FirebaseStorageServices({File image, BuildContext context}) {
```

Comment:
Can't this be a parameter to the methods rather than a constructor?

Reply:
And why you do need BuildContext to upload an image? its bad practice to use context outside the widget tree.

Reply:
Okay i'll fix this too in some time

Reply:
The Scaffold should be used only within a Widget, passing context outside of it sure works, but is very bad as you're separating things that should be in one place.